modules/standard/Path.chpl

```diff
@@ -349,6 +349,9 @@ proc dirname(path: string): string {
   return splitPath(path)[0];
 }
 
+pragma "no doc"
+extern proc sys_getenv(name:c_string, ref string_out:c_string):c_int;
```

Comment:
I think it'd be better to put this declaration as a local declaration to `expandVars`.  By putting it here at the module scope, you're enabling users to call it if they happen to, or discover it since "no doc" doesn't prevent them from accessing it.  By making it a local declaration, you don't even allow them to access it.  Besides making the interface a bit more air-tight, this also has the benefit of reducing the number of symbols that a `use` of this module will see (by epsilon, but in the limit, if all code did this...).

Reply:
It looks like it's only used once, so I think it's fine to do what you've done in that PR, too :)

Reply:
I agree that it seems like preferable style to make declarations local to the scope that needs them when possible.